server/events/project_command_context_builder.go

```diff
@@ -283,9 +284,32 @@ func escapeArgs(args []string) []string {
 	return escaped
 }
 
+/*
+    if len(module.RequiredCore) != 1 {
+		ctx.Log.Info("cannot determine which version to use from terraform configuration, detected %d possibilities.", len(module.RequiredCore))
```

Comment:
Why is this commented out? Please remove if it's not needed

Reply:
Addressed in https://github.com/runatlantis/atlantis/pull/2843/commits/cf69a50bd6fbf3071e11039066a910b24f48556e